examples/ksp/BUILD

```diff
@@ -81,3 +81,73 @@ build_test(
         "//:coffee_app_deploy.jar",
     ],
 )
+
+kt_compiler_plugin(
+    name = "ksp",
```

Comment:
This seems like something to burry inside of `rules_kotlin`, otherwise people might assume it's how they are expected to do things.

Reply:
Is this the only way to provide options for ksp plugins, for example `room.generateKotlin`?